plugins/woocommerce/templates/order/order-details.php

```diff
@@ -82,6 +88,33 @@
 			?>
 		</tbody>
 
+		<?php
+		if ( ! empty( $actions ) ) :
+			?>
+		<tfoot>
+			<tr>
+				<th class="order-actions--heading"><?php esc_html_e( 'Actions', 'woocommerce' ); ?>:</th>
+				<td>
+						<?php
+						$num_actions     = count( $actions );
+						$wp_button_class = wc_wp_theme_get_element_class_name( 'button' ) ? ' ' . wc_wp_theme_get_element_class_name( 'button' ) : '';
+						$i               = 0;
+						foreach ( $actions as $key => $action ) { // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+							if ( empty( $action['aria-label'] ) ) {
+								// Generate the aria-label based on the action name.
+								/* translators: %1$s Action name, %2$s Order number. */
+								$action_aria_label = sprintf( __( '%1$s order number %2$s', 'woocommerce' ), $action['name'], $order->get_order_number() );
+							} else {
+								$action_aria_label = $action['aria-label'];
+							}
+								echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button' . esc_attr( $wp_button_class ) . ' button ' . sanitize_html_class( $key ) . ( ++$i === $num_actions ? '' : ' order-actions--buttons' ) . ' " aria-label="' . esc_attr( $action_aria_label ) . '">' . esc_html( $action['name'] ) . '</a>';
```

Comment:
sorry for another round @alexflorisca but why is `( ++$i === $num_actions ? '' : ' order-actions--buttons' )` needed here? `order-action-buttons` should always be used because the CSS logic gets rid of margin if there's one child using `last-child`. The name of the class is confusing as well because I expect it to be the parents of those buttons, if they have no wrapper then just `order-action-button` is enough, regardless, `$i` and that logic should be removed.

Reply:
Yep right you are, is better with css, made the changes.